lib/round.rb

```diff
@@ -0,0 +1,46 @@
+class Round 
+  attr_accessor :deck, 
+                :turns, 
+                :current_card, 
+                :number_correct, 
+                :take_turn,    
+                :turn_count          
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+    @turn_count = 0
+  end 
+  def current_card
+    self.deck.cards[self.turns.count]
```

Comment:
`self` is redundant here in both places. Ruby convention would say we should not include it.